docker/java/spring/maven-package.sh

```diff
@@ -1,25 +1,29 @@
 #!/usr/bin/env bash
-set -x
+set -xe
 JAVA_AGENT_BUILT_VERSION=${1}
 
 ARTIFACT_ID=elastic-apm-agent
 
 if [ -z "${JAVA_AGENT_BUILT_VERSION}" ] ; then
   cd /agent/apm-agent-java
-  mvn -q --batch-mode install -DskipTests \
+  git log -1
+  mvn -q --batch-mode clean install -DskipTests=true -Dhttps.protocols=TLSv1.2 -Dmaven.javadoc.skip=true \
```

Comment:
Using the mvn wrapper
```suggestion
  ./mvnw -q --batch-mode clean install -DskipTests=true -Dhttps.protocols=TLSv1.2 -Dmaven.javadoc.skip=true \
```

Reply:
the wrapper needs the `-Dmaven.repo.local=.m2` setting to work we are using a maven Docker image and the have to use maven on some occasions because the wrapper is not available, I thinking to go back to use the regular maven that works about of the box